docs/website/mmp-errors.md

```diff
@@ -248,6 +250,15 @@ To remove the warning either remove the optimization argument to mmp, or pass
 By default this option will be automatically enabled whenever it's possible
 and safe to do so.
 
+### <a name="MM2108"/>MM2108: '{0}' was stripped of architecture '{1}' to comply with App Store restrictions. This could break exisiting codesigning signatures. Consider stripping the library with lipo or disabling with --optimize=-trim-architectures");
+
+The App Store now rejects applications which contain libraries and frameworks which contain 32-bit variants. The library was stripped of unused archtectures when copied into the final application bundle.
+
+This is in general safe, and will reduce application size as an added benefit, but may break existing codesigning signatures if it was already signed.
```

Comment:
We do, but if there was some pre-existing signature on the frameowork we're copying we'd be breaking that and resigning over top.

Maybe it isn't worth mentioning that corner case. In 99% of use cases we're going to fix things when signing at the end.

Reply:
My point is that if we're always signing dylibs and frameworks, stripping unused architectures from them won't change anything (since we always overwrite any existing signature anyway).

Reply:
Ah, I see, it's not required to sign the .app like it is for XI, in which case this does make sense. So nvm.